README.md

```diff
@@ -5,7 +5,20 @@ the [EUDI Wallet Reference Implementation project description](https://github.co
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0)
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 15.2.5.
+## Table of contents
+
+* [Overview](#overview)
+* [Development server](#development-server)
+* [Code scaffolding](#code-scaffolding)
+* [Build](#build)
+* [How to run](#how-to-run)
+* [Running tests](#running-tests)
+* [License](#license)
+
+## Overview
+
+This is a WEB UI that provides functionality to interact with the Verifier/RP trusted end-point implemented [here](https://github.com/eu-digital-identity-wallet/eudi-srv-web-verifier-endpoint-23220-4-kt).
```

Comment:
Perhaps here we could also add an alternative description:

Another way to think of this application is that it represents an arbitrary application that wants to delegate to the [trusted end-point](https://github.com/eu-digital-identity-wallet/eudi-srv-web-verifier-endpoint-23220-4-kt) the burden of interacting with a wallet using OpenId4VP

Reply:
Added test to overview